src/pivot-table/data/extract-headers.ts

```diff
@@ -36,6 +39,9 @@ const extractHeaders = (
         fieldId: qDimensionInfo.qGroupFieldDefs[qDimensionInfo.qGroupPos],
         isActivelySorted: colIdx === (hyperCube.activelySortedColumn?.colIdx ?? 0),
         isLocked: qDimensionInfo.qLocked ?? false,
+        label: "",
```

Comment:
This should be the same value as `title`, but since the HeaderMenu is dependent on this I see no value in keeping two idential values here. Remove `title` and use `label` instead?